tests_config/run_spec.js

```diff
@@ -56,20 +45,22 @@ function run_spec(dirname, options, additionalParsers) {
 
       if (RUN_AST_TESTS) {
         const source = read(dirname + "/" + filename);
-        const ast = removeEmptyStatements(parse(source));
-        let ppast;
+        const ast = parse(source, mergedOptions);
+        const astMassaged = massageAST(ast);
```

Comment:
Heh, I got ahead of myself :P It should be good now, along with the Node 4 test.